doc/rofi-dmenu.5

```diff
@@ -147,6 +147,13 @@ Or any combination: '5,-3:,7:11,2,0,-9'
 .PP
 Urgent row, mark \fIX\fP as urgent. See \fB\fC-a\fR option for details.
 
+.PP
+\fB\fC-select-rows\fR \fIX\fP
```

Comment:
The term "select" can be confusing since it already used to mean highlighting the current item in the list before either confirm selection or check the box.

Then again, Rofi has already name the flag `-multi-select` so it could be that the maintainers do care about this term.

Reply:
It is the term that is used everywhere else.. Using another term here would be confusing.